pkg/webhook/server.go

```diff
@@ -82,6 +86,74 @@ func (s *Server) setDefaults() {
 	}
 }
 
+// For registers defaulting and validation webhooks for the provided types that implement either
+// webhooktuil.validator or webhookutil.defaulter.
+func (s *Server) EnableDefaultsFor(scheme *runtime.Scheme, objects ...runtime.Object) error {
+	// Create a webhook for each type
+	for _, object := range objects {
+		wh := newMutatingWebhookFor(object)
+		if wh == nil {
+			continue
+		}
+
+		gvks, _, err := scheme.ObjectKinds(object)
+		if err != nil {
+			return err
+		}
+		if len(gvks) != 1 {
+			return fmt.Errorf("expected only GVK returned by scheme.ObjectKinds")
+		}
+		fmt.Println("/mutate-" + generatePath(gvks[0]))
```

Comment:
logging this info at some verbosity level may not be a bad idea, can help in debugging.

Reply:
My debugging code sneaked in.
I agree it can be converted to logging.
Will fix it